src/server/routes/api/todos.js

```diff
@@ -55,4 +55,24 @@ router.put("/delete", (req, res) => {
     .catch(err => res.status(400).json('Error' + err));
 });
 
+// @route PUT api/todos/edit
+// @desc Edit todo items
+// @access Public
+router.put("/edit", (req, res) => {
+  // Grab the userId from the query
+  const { id, todoTitle } = req.query;
+
+  // Search Todos collection for all todo items for username
+
+  Todo.findByIdAndUpdate(
+    id,
+    {
+      todoItem: {
```

Comment:
Have you tested that this works? I thought the syntax for updating an inner documents is:

```
$set: {
        'todoItem.title': todoTitle
      }
```

I could be wrong, I'll have to test.